modin/pandas/series.py

```diff
@@ -1267,23 +1333,40 @@ def nsmallest(self, n=5, keep="first"):  # noqa: PR01, RT01, D200
         )
 
     def shift(
-        self, periods=1, freq=None, axis=0, fill_value=None
+        self,
+        periods=1,
+        freq=None,
+        axis=0,
+        fill_value=lib.no_default,
+        suffix=None,
     ):  # noqa: PR01, RT01, D200
         """
         Shift index by desired number of periods with an optional time `freq`.
         """
+        # pandas 2.1.0rc0 ignores suffix parameter
```

Comment:
is this a design decision or a bug in pandas?

Reply:
I'm not sure right now and create the issue for that: https://github.com/pandas-dev/pandas/issues/54806

Reply:
Seems like `suffix` will eventually be removed (according to their reaction to your issue). We'll just have to keep our API synced 😸